File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/models/youtubeSearch/Id.java
package com.github.kotvertolet.youtubeaudioplayer.data.models.youtubeSearch;

import com.google.gson.annotations.SerializedName;

public class Id {

    @SerializedName("kind")
    private String kind;

    @SerializedName("videoId")
    private String videoId;

    public Id(String videoId) {
        this.videoId = videoId;
    }

    public Id(String kind, String videoId) {
        this.kind = kind;
        this.videoId = videoId;
    }

    public String getKind() {
        return kind;
    }

    public void setKind(String kind) {
        this.kind = kind;
    }

    public String getVideoId() {
        return videoId;
    }

    public void setVideoId(String videoId) {
        this.videoId = videoId;
    }

    @Override
    public String toString() {
        return
                "Id{" +
                        "kind = '" + kind + '\'' +
                        ",videoId = '" + videoId + '\'' +
                        "}";
    }
}

File: app/src/main/java/com/github/kotvertolet/youtubeaudioplayer/data/models/youtube/videos/list/Id.kt
package com.github.kotvertolet.youtubeaudioplayer.data.models.youtube.videos.list

import com.google.gson.annotations.SerializedName

class Id {
    @SerializedName("kind")
    var kind: String? = null

    @SerializedName("videoId")
    var videoId: String

    constructor(videoId: String) {
        this.videoId = videoId
    }

    constructor(kind: String?, videoId: String) {
        this.kind = kind
        this.videoId = videoId
    }

    override fun toString(): String {
        return "Id{" +
                "kind = '" + kind + '\'' +
                ",videoId = '" + videoId + '\'' +
                "}"
    }
}